multiapps-controller-persistence/src/main/java/module-info.java

```diff
@@ -2,20 +2,28 @@
 
     exports org.cloudfoundry.multiapps.controller.persistence;
     exports org.cloudfoundry.multiapps.controller.persistence.dialects;
+    exports org.cloudfoundry.multiapps.controller.persistence.dto;
     exports org.cloudfoundry.multiapps.controller.persistence.jclouds.providers.aliyun;
     exports org.cloudfoundry.multiapps.controller.persistence.jclouds.providers.aliyun.blobstore;
     exports org.cloudfoundry.multiapps.controller.persistence.jclouds.providers.aliyun.blobstore.config;
     exports org.cloudfoundry.multiapps.controller.persistence.model;
+    exports org.cloudfoundry.multiapps.controller.persistence.model.adapter;
+    exports org.cloudfoundry.multiapps.controller.persistence.model.filters;
     exports org.cloudfoundry.multiapps.controller.persistence.query;
+    exports org.cloudfoundry.multiapps.controller.persistence.query.criteria;
+    exports org.cloudfoundry.multiapps.controller.persistence.query.impl;
```

Comment:
Should any `impl` packages be exported?

Reply:
They shouldn't be, but unfortunately we have to export this one, since it's used in the xsa-multiapps-controller project (see `CredentialStoreConfigurationEntryQueryImpl`).